js_modules/dagster-ui/packages/ui-core/src/instance/backfill/BackfillPartitionsRequestedDialog.tsx

```diff
@@ -1,57 +1,91 @@
-import {Button, Dialog, DialogFooter, FontFamily} from '@dagster-io/ui-components';
+import {Box, Button, Dialog, DialogFooter, FontFamily, Spinner} from '@dagster-io/ui-components';
 import {useMemo} from 'react';
 
-import {BackfillTableFragment} from './types/BackfillTable.types';
+import {gql, useQuery} from '../../apollo-client';
+import {
+  BackfillPartitionsDialogContentQuery,
+  BackfillPartitionsDialogContentQueryVariables,
+} from './types/BackfillPartitionsRequestedDialog.types';
 import {TruncatedTextWithFullTextOnHover} from '../../nav/getLeftNavItemsForOption';
 import {VirtualizedItemListForDialog} from '../../ui/VirtualizedItemListForDialog';
 
 const COLLATOR = new Intl.Collator(navigator.language, {sensitivity: 'base', numeric: true});
+
 interface Props {
-  backfill?: BackfillTableFragment;
+  backfillId?: string;
   onClose: () => void;
 }
-export const BackfillPartitionsRequestedDialog = ({backfill, onClose}: Props) => {
+
+export const BackfillPartitionsRequestedDialog = ({backfillId, onClose}: Props) => {
   return (
     <Dialog
-      isOpen={!!backfill}
+      isOpen={!!backfillId}
       title={
         <span>
           Partitions requested for backfill:{' '}
-          <span style={{fontSize: '16px', fontFamily: FontFamily.monospace}}>{backfill?.id}</span>
+          <span style={{fontSize: '16px', fontFamily: FontFamily.monospace}}>{backfillId}</span>
```

Comment:
I don't think this should need a `fontSize` on it, ideally the default/mono mismatched font size problem went away with Geist.

Reply:
Ahh can confirm, looks the same without it! Looks like this is one of the only places we specify fontSize: 16px in the whole codebase now. 

![image](https://github.com/user-attachments/assets/4d64d333-868f-494c-a3d6-76e48619216d)
![image](https://github.com/user-attachments/assets/c46e1262-612c-4ee7-9f2f-b9df4784dad5)